tests/software/minia/main.nf

```diff
@@ -7,7 +7,7 @@ include { MINIA } from '../../../software/minia/main.nf' addParams( options: [:]
 workflow test_minia {
     input = [ [ id:'test' ], // meta map
               [file("${launchDir}/tests/data/genomics/sarscov2/illumina/fastq/test_1.fastq.gz", checkIfExists: true),
-              file("${launchDir}/tests/data/genomics/sarscov2/illumina/fastq/test_2.fastq.gz", checkIfExists: true)] 
+              file("${launchDir}/tests/data/genomics/sarscov2/illumina/fastq/test_2.fastq.gz", checkIfExists: true)]
```

Comment:
This hasn't been updated?

Reply:
oh shit sorry ... there was some other stuff in my git history it seems :facepalm: 
only the last two commits were actually intentional.
damnit ...

Reply:
well I just updated it anway